src/Parallel/NodeLock.hpp

```diff
@@ -8,66 +8,93 @@
 #include "Utilities/Gsl.hpp"
 #include "Utilities/NoSuchType.hpp"
 
+#include "Parallel/Printf.hpp"
```

Comment:
In addition to the one @nilsdeppe found in the test -- is this also left over from debugging?